doc/source/_templates/layout.html

```diff
@@ -1,4 +1,8 @@
 {% extends "!layout.html" %}
 
-{% block relbar2 %}{% endblock %}
+{% block extrahead %}
+<link href='http://fonts.googleapis.com/css?family=Open+Sans:400italic,700italic,400,700' rel='stylesheet' type='text/css'>
```

Comment:
I'm happy to go the more general route if you know how to set it up. Maybe you can get it in a subsequent PR?